terraform/deployments/create-or-update/main.tf

```diff
@@ -34,9 +34,15 @@ resource "azurerm_nginx_deployment" "example" {
   }
 
   frontend_public {
-    ip_address = [module.prerequisites.public_ip_address_id]
+    ip_address = [module.prerequisites.public_ipv4_address_id]
+    # Use following to enable the frontend only with an ipv6 address
+    #ip_address = [module.prerequisites.public_ipv6_address_id]
```

Comment:
Maybe also mention that the prerequisites block also needs to be uncommented.

Reply:
Done